src/Entity/Shipment.php

```diff
@@ -95,9 +95,14 @@ class Shipment
     private $movementReferenceNumber;
 
     /**
-     * @var InvoiceLineTotal
+     * @var invoiceLineTotal
      */
     private $invoiceLineTotal;
+    
+    /**
+     * @var shipmentTotalWeight
```

Comment:
Same here, typehint probably wrong name.

Reply:
The class name is capitalized and I have it capitalized in shipment. I'm not sure what your asking for here.